resources/assets/js/components/search/search.js

```diff
@@ -0,0 +1,7 @@
+$( "#global_search_form").submit(function( event ) {
+  var search_val = $('#global_search_input').val();
```

Comment:
Identifier 'search_val' is not in camel case.<br>Mixed double and single quotes.